### DISCUSSION TITLE
New installer detected as a trojan

### DISCUSSION THREAD

**Discussion by zer09** [wezterm/wezterm#4879] [General]
I installed `wezterm` using winget and got an update notification today, but the winget repo is still not updated so I download the installer from the website and got flag by windows defender that it is a virus.

![image](https://github.com/wez/wezterm/assets/780780/eba02377-0ec7-4a43-bd3e-f5123ddc136d)

**Comment by wez** [comment]
That happens from time to time; see https://github.com/wez/wezterm/issues/3980
Could you submit a false positive to Defender? microsoft.com refuses to let me sign in with my personal identity to do it myself.
https://www.microsoft.com/en-us/wdsi/filesubmission

**Reply by zer09** [reply]
Submitted
![image](https://github.com/wez/wezterm/assets/780780/d71e28a1-b52b-49c1-bb24-f1eea97aade3)

**Reply by wez** [reply]
Thanks!
